include/z64message.h

```diff
@@ -82,6 +82,70 @@ typedef enum {
     /* 0x37 */ MSGMODE_PAUSED // Causes the message system to do nothing until external code sets a new message mode or calls a public function
 } MessageMode;
 
+typedef enum FaceReactionSet {
+    /* 0x00 */ FACE_REACTION_SET_CARPENTER_BOSS,
+    /* 0x01 */ FACE_REACTION_SET_CARPENTER_1,
+    /* 0x02 */ FACE_REACTION_SET_CARPENTER_2,
+    /* 0x03 */ FACE_REACTION_SET_CARPENTER_3,
+    /* 0x04 */ FACE_REACTION_SET_CARPENTER_4,
+    /* 0x05 */ FACE_REACTION_SET_HYRULIAN_GUARD,
+    /* 0x06 */ FACE_REACTION_SET_HEISHI4_1,
+    /* 0x07 */ FACE_REACTION_SET_HEISHI4_2,
+    /* 0x08 */ FACE_REACTION_SET_CUCCO_LADY,
+    /* 0x09 */ FACE_REACTION_SET_CARPENTERS_SON,
+    /* 0x0A */ FACE_REACTION_SET_KAKARIKO_ROOF_MAN,
+    /* 0x0B */ FACE_REACTION_SET_WINDMILL_MAN,
+    /* 0x0C */ FACE_REACTION_SET_12, // Unused
+    /* 0x0D */ FACE_REACTION_SET_CURSED_SKULLTULA_MAN,
+    /* 0x0E */ FACE_REACTION_SET_DAMPE,
+    /* 0x0F */ FACE_REACTION_SET_GRAVEYARD_KID,
+    /* 0x10 */ FACE_REACTION_SET_SARIA,
+    /* 0x11 */ FACE_REACTION_SET_MIDO,
+    /* 0x12 */ FACE_REACTION_SET_FADO,
+    /* 0x13 */ FACE_REACTION_SET_KOKIRI_CHILD_1,
+    /* 0x14 */ FACE_REACTION_SET_KOKIRI_CHILD_2,
+    /* 0x15 */ FACE_REACTION_SET_SKULL_KID,
+    /* 0x16 */ FACE_REACTION_SET_CHILD_ZELDA,
```

Comment:
The same way you didn't specify for ruto and malon I think you can omit "child"
(I don't see how you can get to zelda with a mask anyway?)